backstop/tests/table.html

```diff
@@ -1775,104 +1781,134 @@ <h2>Additional options</h2>
         style="max-height: 407px"
         id="demo-extras-scrollable-table"
       >
-        <div class="iui-table-header">
-          <div class="iui-row">
-            <div class="iui-cell iui-actionable iui-sorted">
-              Product
-              <button
-                class="iui-button iui-borderless iui-filter-button"
-                aria-label="Apply filter"
-                type="button"
+        <div class="iui-table-header-wrapper">
+          <div class="iui-table-header">
+            <div class="iui-row">
+              <div
+                class="iui-cell iui-slot iui-sticky"
+                style="left: 0;"
               >
-                <svg-filter-hollow
-                  class="iui-button-icon"
-                  aria-hidden="true"
-                ></svg-filter-hollow>
-              </button>
-              <div class="iui-cell-end-icon">
-                <svg-sort-up
-                  class="iui-button-icon iui-sort"
-                  aria-hidden="true"
-                ></svg-sort-up>
+                <input
+                  class="iui-checkbox"
+                  type="checkbox"
+                />
               </div>
-              <div class="iui-resizer">
-                <div class="iui-resizer-bar"></div>
-              </div>
-            </div>
-            <div class="iui-cell iui-actionable">
-              Price
-              <button
-                class="iui-button iui-borderless iui-filter-button"
-                aria-label="Apply filter"
-                type="button"
+              <div
+                class="iui-cell iui-actionable iui-sorted iui-sticky"
+                style="left: 48px;"
```

Comment:
How are these inline styles supposed to work? We need to apply them in react too?

What if the user wants the first 3 columns to be sticky? We need to find their widths using JS?

And if we really need to pass those from JS then maybe we should use CSS variables.

Reply:
Yup, those needed to be set through JS. I also thought about using CSS variables but I am not sure what the benefit of using CSS variables here would be.

Reply:
We try to avoid inline styles in react even if there is no obvious benefit.

Same reason we use `--left` instead of `left` in slider and color-picker

Reply:
Using CSS variables right now